<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OPNWidget</class>
 <widget class="QWidget" name="OPNWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="OPNOperatorWidget" name="operatorWidget1" native="true"/>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="FMLFOWidget" name="lfoWidget" native="true"/>
   </item>
   <item row="0" column="3" rowspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="4">
    <widget class="AlgorithmWidget" name="algorithmWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="OPNOperatorWidget" name="operatorWidget2" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="OPNOperatorWidget" name="operatorWidget3" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="OPNOperatorWidget" name="operatorWidget4" native="true"/>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="PianoWidget" name="pianoWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>192</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AlgorithmWidget</class>
   <extends>QWidget</extends>
   <header>opn/algorithmwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PianoWidget</class>
   <extends>QWidget</extends>
   <header>common/pianowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FMLFOWidget</class>
   <extends>QWidget</extends>
   <header>opn/fmlfowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OPNOperatorWidget</class>
   <extends>QWidget</extends>
   <header>opn/opnoperatorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
